sanic/response.py

```diff
@@ -53,7 +54,7 @@ def __init__(self, streaming_fn, status=200, headers=None,
         self.content_type = content_type
         self.streaming_fn = streaming_fn
         self.status = status
-        self.headers = headers or {}
+        self.headers = CIMultiDict(headers or {})
```

Comment:
I know you didn't do this, but would it make sense to just make `{}` the default for `headers`, instead of `None`?

Reply:
No, python default argument is mutable, making `{}` default is a bad pattern. @r0fls 

http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments